### PR TITLE
deepin.dde-control-center: init at 5.6.3

### PR DESCRIPTION
**Pull request by wineee**
###### Description of changes
https://github.com/linuxdeepin/dde-nixos/issues/9

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/desktops/deepin/core/dde-control-center/default.nix [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+  };
+
+  patches = [
+    (fetchpatch {
+      name = "fix info for other distributions";
+      url = "https://github.com/linuxdeepin/dde-control-center/commit/32394aa84f4b575e0a84a0813ba07b72cb1ba137.patch";
+      sha256 = "sha256-r21oczFyhKarMuEkL8Ruzd8jqB/T+MfuUGrLNeQdZB8=";
+    })
+    ./fix-svg-render-for-themeitem.patch
+    ./dont-show-endUserLicenseAgreement-for-deepinos.patch
```
[the remark this answers is not in the document]
Comment (wineee) [reply pkgs/desktops/deepin/core/dde-control-center/default.nix:55]: Should I replace the text？I don't find `endUserLicenseAgreement` for NixOS,

[the remark this answers is not in the document]
Comment (wineee) [reply pkgs/desktops/deepin/core/dde-control-center/default.nix:50]: `User-Experience-Program-License-Agreement-Community.md ` comes from a non-open source component（deepin-deepinid-client），It's only blank page if we didn't install deepin-deepinid-client

dde-control-center 6.0.0 is completely rewritten，not based on 5.6.*, so this patch cannot merge by upstream

[the remark this answers is not in the document]
Comment (SuperSandro2000) [reply pkgs/desktops/deepin/core/dde-control-center/default.nix:50]: Can you add a short comment with that info and a link to the upstream commit that removed it?

[the remark this answers is not in the document]
Comment (wineee) [reply pkgs/desktops/deepin/core/dde-control-center/default.nix:53]: Comment has been added. dde-control-center 6.0.0 is not a simple version upgrade, but a rewrite，this patch will remove in future

**Pull request merged**